test/evergreen.yml

```diff
@@ -1586,6 +1586,26 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: spinlock-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
```

Comment:
Also I think it's better to have a separate compile task so that we can run make check and python unit tests in parallel.